### PR TITLE
Catch exceptions when creating/deleting clusters

### PR DESCRIPTION
**Pull request by giorgiosav**
Catches exceptions when deleting or creating clusters and tells user to run `gcloud container clusters list`. This addresses the strange behavior described in #94, which seems to be an issue with Gcloud and not MLBench.

(I tried to add tests for this in `tests/test_cli.py` but for some reason I couldn't get pytest to recognize the `sys.exit`...)

### CODE REVIEW

File: mlbench_core/cli/cli.py [hunk 1]
```diff
@@ -386,8 +387,16 @@ def delete_gcloud(name, zone, project):
     name_path = "projects/{}/locations/{}/".format(project, zone)
 
     cluster_path = "{}clusters/{}".format(name_path, name)
-
-    response = gclient.delete_cluster(None, None, None, name=cluster_path)
+    try:
+        response = gclient.delete_cluster(None, None, None, name=cluster_path)
+    except NotFound as e:
+        click.echo("Exception from Google:")
+        print(e)
```
Comment (Panaetius) [inline mlbench_core/cli/cli.py:394]: Why not use click.echo?

File: mlbench_core/cli/cli.py [hunk 2]
```diff
@@ -515,7 +525,18 @@ def create_gcloud(
         logging_service=None,
         monitoring_service=None,
     )
-    response = gclient.create_cluster(None, None, cluster, parent=name_path)
+    try:
+        response = gclient.create_cluster(None, None, cluster, parent=name_path)
+    except AlreadyExists as e:
+        click.echo("Exception from Google:")
+        print(e)
```
Comment (Panaetius) [inline mlbench_core/cli/cli.py:532]: Same as above

**Comment by ehoelzl** [comment]
@giorgiosav Update on this PR ?

**Comment by giorgiosav** [comment]
> @giorgiosav Update on this PR ?
Sorry, I changed `print` with `click.echo`

**Comment by giorgiosav** [comment]
> @giorgiosav Update on this PR ?

I tried to implement the changes but because this is an old PR it is creating a lot of conflicts... I think it's best to close it and reimplement the changes on a newer branch.

**Comment by ehoelzl** [comment]
@giorgiosav Okay no problem. Can you take care of it ? If not that's okay, I can integrate the ghanges

**Comment by giorgiosav** [comment]
Sure, I can do it

**Pull request closed without merging**